flycheck.el

```diff
@@ -1945,9 +1970,26 @@ nil otherwise."
                                 (funcall enabled)))))
     (if shall-enable
         (cl-pushnew checker flycheck-enabled-checkers)
```

Comment:
Don't we want to rename this to `flycheck-automatically-enabled-checkers`, for symmetry? (We'll need an alias, since https://github.com/search?q=flycheck-enabled-checkers&type=Code some people seem to use that variable?!)

Reply:
Yes, I agree that it makes sense for the symmetry.  I didn't do it precisely because I didn't want to break the API.

Maybe we add an alias and deprecate the former one?

Reply:
An alias sounds good.